src/Commands/ClearPendingTenants.php

```diff
@@ -27,8 +27,8 @@ public function handle(): int
         $olderThanHours = (int) $this->option('older-than-hours');
 
         if ($olderThanDays && $olderThanHours) {
-            $this->line("<options=bold,reverse;fg=red> Cannot use '--older-than-days' and '--older-than-hours' together \n"); // todo@cli refactor all of these styled command outputs to use $this->components
-            $this->line('Please, choose only one of these options.');
+            $this->components->error("Cannot use '--older-than-days' and '--older-than-hours' together");
+            $this->components->error('Please, choose only one of these options.');
```

Comment:
```suggestion
            $this->components->error("Cannot use '--older-than-days' and '--older-than-hours' together. Please, choose only one of these options.");
```

This is just a small detail, but I don't like displaying "Please, choose..." as a separate error because it isn't an error. Maybe there's a better `$this->components` method for this (maybe `info` would be okay?), but I think merging these lines would be even better

Reply:
Merging them looks better